dataprofiler/tests/profilers/profiler_options/test_unstructured_text_profile_options.py

```diff
@@ -244,3 +247,57 @@ def test_eq(self):
         self.assertNotEqual(options, options2)
         options2.words.is_enabled = False
         self.assertEqual(options, options2)
+
+    def test_json_encode(self):
```

Comment:
something like this could become:
```python
     def test_json_encode(self):
        option = TextProfilerOptions(
            is_enabled=False,
            is_case_sensitive=False,
            stop_words={"ab", "aa", "aba", "aa"},
            top_k_chars=5,
            top_k_words=8,
        )
        serialized = json.dumps(option, cls=ProfileEncoder)
        expected_json = {
            "class": "TextProfilerOptions",
            "data": {
                "is_enabled": False,
                "is_case_sensitive": False,
                "stop_words": ["ab", "aba", "aa"],
                "top_k_chars": 5,
                "top_k_words": 8,
                "vocab": {"class": "BooleanOption", "data": {"is_enabled": True}},
                "words": {"class": "BooleanOption", "data": {"is_enabled": True}},
            }
        }

        actual_option_json = json.loads(serialized)
        self.assertDictEqual(expected_json, actual_option_json)
```
given the above case

Reply:
I think for this since the nested is just the BooleanOption, it isn't a major deal to write out.